src/watch/index.ts

```diff
@@ -48,9 +48,9 @@ export class Watcher {
 
 	close() {
 		if (this.buildTimeout) clearTimeout(this.buildTimeout);
-		this.tasks.forEach(task => {
+		for (const task of this.tasks) {
```

Comment:
I replaced some .forEach with more efficient for loops, not really related to the task but who know when I will have a look at these files the next time.